x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/types.ts

```diff
@@ -20,7 +20,6 @@ export interface IFlashMessage {
 // @see EuiGlobalToastListToast for more props
 export interface ToastOptions {
   iconType?: string;
-  id?: string;
```

Comment:
The notifications toast do not support setting a custom id, but I couldn't find any usages of this so I removed it, :tada: for types